collections/src/main/scala/strawman/collection/IterableOnce.scala

```diff
@@ -135,3 +138,831 @@ object IterableOnce {
   @`inline` implicit def iterableOnceExtensionMethods[A](it: IterableOnce[A]): IterableOnceExtensionMethods[A] =
     new IterableOnceExtensionMethods[A](it)
 }
+
+/** This implementation trait can be mixed into an `IterableOnce` to get the basic methods that are shared between
+  * `Iterator` and `Iterable`. The `IterableOnce` must support multiple calls to `iterator()` but may or may not
+  * return the same `Iterator` every time.
+  */
```

Comment:
You should define the documentation variables that are not inherited but used in the operations’ scaladoc.